### DISCUSSION TITLE
Cylc 8 GUI on Monsoon

### DISCUSSION THREAD

**Discussion by dennissergeev** [MetOffice/simulation-systems#200] [Monsoon]
Hello,

Is it possible to run Cylc 8 in a GUI mode on Monsoon? I suspect it isn't but I thought I'd check.

Cheers,
Denis

**Comment by jennyhickson** [comment]
Hi Denis, 

I think its not - I've been recommending use of cylc tui as an alternative. 

Cheers
Jenny

**Reply by dennissergeev** [reply]
Thanks Jenny! Cylc TUI is very nice but doesn't seem to have the same functionality as the Cylc 7 GUI. For example, I would like to reset tasks or skip them - it it even possible in the TUI?

**Comment by mo-roger-milton** [comment]
Unfortunately the Cylc 8 GUI is not available on the current generation of Met Office HPCs, this is an issue that will be fixed once we are on the new machine later this year :-) 
There is some information on this in the Monsoon User Guide, see https://code.metoffice.gov.uk/doc/monsoon2/roseCylc.html#cylc-8

**Reply by dennissergeev** [reply]
Thanks Roger! Sorry I missed that in the user guide. Looking forward to using the new machine!

**Reply by mo-roger-milton** [reply]
No probs, just pointing out for future readers. 

We're all looking forward to the new machine!
